src/core/city-autocomplete.service.js

```diff
@@ -35,7 +36,8 @@ function CityAutocomplete($q, $http, appConfig, UdbPlace, jsonLDLangFilter) {
         'disableDefaultFilters': true,
         'embed': true,
         'limit': 1000,
-        'sort[created]': 'asc'
+        'sort[created]': 'asc',
+        'text': '*' + textParam + '*'
```

Comment:
Have you tested how SAPI3 would handle this with an empty string? I know it's not possible from the UI at this point because it only gets called once the user enters 3+ characters, but you do default the argument to an empty string so it could happen at some point if this function gets used someplace else in the future.

Reply:
Might be safer to not include the text parameter at all if it's `**` 🤔